libimage/containers.conf

```diff
@@ -0,0 +1,5 @@
+[engine]
+# Default location for storing temporary container image content. Can be overridden with the TMPDIR environment
+# variable. If you specify "storage", then the location of the
+# container/storage tmp directory will be used.
+image_copy_tmp_dir="/tmp/from/containers.conf"
```

Comment:
This file doesn't seem to be used.  Test files should go into `libimage/testdata`.

Reply:
Yes I could not get this to work, because the test program had preloaded the containers.conf, and will not reload.

Reply:
Did you try via CONTAINERS_CONF=...?

Reply:
I tried to set it while in the tests, and it was ignored. Outside of the test with my test program everything worked fine.

Reply:
Ok I figured it out.